webrtc.html

```diff
@@ -1807,16 +1807,20 @@
                         </ol>
                       </li>
                       <li>
-                        <p>If <code><var>candidate</var>.ufrag</code> is not
-                        equal to any ufrag present in the corresponding
-                        <a>media description</a> of an applied remote
-                        description, reject <var>p</var> with an
+                        <p>If <code><var>candidate</var>.ufrag</code> is
+                        non-null, and is not equal to any ufrag present in the
```

Comment:
In IDL terms, the members of dictionaries are "missing" when they are not there. Since the `ufrag` dictionary member is of type DOMString and not nullable, must be a string if it is there at all.